running-code/execute-script.md

```diff
@@ -0,0 +1,191 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(execute-script)=
+# Execute a Python script
```

Comment:
```suggestion
# How to Execute a Python Script
```
I think we want to use title case for h1 titles and then sentence case for h2 --> h6